site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -79,11 +79,69 @@ spec:
       seconds: 30
 ```
 
+{{% feature publishVersion="1.37.0" %}}
+Counter-based `FleetAutoscaler` specification below and in the {{< ghlink href="examples/counterfleetautoscaler.yaml" >}}example folder{{< /ghlink >}}:
+
+```yaml
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: fleet-autoscaler-counter
+spec:
+  fleetName: fleet-example
+  policy:
+    type: Counter  # Counter based autoscaling
+    counter:
+      # Key is the name of the Counter. Required field.
+      key: players
+      # BufferSize is the size of a buffer of counted items that are available in the Fleet (available capacity).
+      # Value can be an absolute number (ex: 5) or a percentage of the Counter available capacity (ex: 5%). 
+      # An absolute number is calculated from percentage by rounding up. Must be bigger than 0. Required field.
+      bufferSize: 5
+      # MinCapacity is the minimum aggregate Counter total capacity across the fleet.
+      # If zero, MinCapacity is ignored. 
```

Comment:
Same comments as left in the examples/listfleetautoscaler.yaml and examples/counterfleetautoscaler.yaml.